odxtools/cli/browse.py

```diff
@@ -99,7 +99,7 @@ def prompt_single_parameter_value(parameter: Parameter) -> Optional[AtomicOdxTyp
         return None
     elif parameter.physical_type.base_data_type is not None:
         return _convert_string_to_odx_type(
-            answer.get(parameter.short_name), parameter.physical_type.base_data_type)
+            cast(str, answer.get(parameter.short_name)), parameter.physical_type.base_data_type)
```

Comment:
```suggestion
            str(answer.get(parameter.short_name)), parameter.physical_type.base_data_type)
```

Reply:
not sure: without the cast mypy has the following complaint:
```
odxtools/cli/browse.py:102: error: Argument 1 to "_convert_string_to_odx_type" has incompatible type "str | bool | list[Any] | None"; expected "str"  [arg-type]
```

I guess it is better to make it shut up and deal with the exception if something goes wrong anyway instead of making it "work" unconditionally?